packages/riverpod/test/framework/provider_element_test.dart

```diff
@@ -1281,6 +1281,24 @@ void main() {
     });
   });
 
+  group('ref.onReload', () {
+    test('is called on refresh', () {
```

Comment:
Could you test WidgetRef.refresh and Ref.refresh too? Currently only ProviderContainer.refresh is tested

Reply:
I've added test for Ref and WidgetRef - the `Ref` test is a bit more verbose, as it also tries to test forwarding the refresh call to a dependency. hope thats fine.